content/guides/user-guides/delfin/dashboard/_index.md

```diff
@@ -0,0 +1,170 @@
+---
+title: SRM using SODA Dashboard
+description: "User guide for Monitoring through Delfin integrated SODA Dashboard"
+weight: 10
+disableToc: false
+tags: ["user guide", "delfin", "dashboard"] 
+---
+
+delfin (Dolphin in Spanish!), the SODA Infrastructure Manager project is an open source project to provide unified, intelligent and scalable resource management, alert and performance monitoring. It covers the resource management of all the storage backends & other infrastructures under SODA deployment. It also provides the alert management and metric data(performance/health) for monitoring and further analysis.
+
+It provides unified APIs to access, export and connect with clients as well as a set of interfaces for various driver additions. These APIs combined with the SODA Dashboard provide a graphical interface that allows users to manage storage backends and visualize the metrics and health.
+
+### Delfin Features
+- [Register a Storage Device](#register-storage-device)
+    - [REST Access](#rest-access)
+    - [SSH Access](#ssh-access)
+- [Storage Summary](#storage-summary)
+- [Storage Device Actions](#storage-device-actions)
+    - Update access Info
+    - Configure Alert Source
+    - Remove Alert Source
+    - Sync Storage Device
+    - Configure Metric collection
+    - Delete Storage Device
+- [Storage Device Details](#storage-device-details)
+- [Performance Monitor](#performance-monitor)
+- [Alertmanager](#alertmanager)
```

Comment:
Update to "Alert Manager"

Reply:
Alertmanager is as per the [official docs](https://prometheus.io/docs/alerting/latest/alertmanager/)

Reply:
ok